app/helpers/smart_proxies_helper.rb

```diff
@@ -1,5 +1,5 @@
 module SmartProxiesHelper
-  TABBED_FEATURES = ["Puppet", "Puppet CA", "Logs"]
+  TABBED_FEATURES = ["Puppet CA", "Logs"]
```

Comment:
It's weird to me that Puppet is removed but Puppet CA stays. Especially if assignment of hosts to a Puppetserver will remain.

Reply:
Agreed it is weird, that's because it is not finished, both Puppet and Puppet CA should be extracted, according to my last info.

- https://github.com/theforeman/foreman_puppet/pull/146
- https://github.com/ezr-ondrej/foreman/pull/62